style.css

```diff
@@ -0,0 +1,914 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Lato", sans-serif;
+}
+
+.nav-container {
+  text-decoration: none;
+  width: 100%;
+}
+
+.top-hader {
+  display: none;
+}
+
+.header2 {
+  background-color: rgba(255, 255, 255, 0.168);
+}
+
+.rotate {
+  display: none;
+}
+
+.humburger {
+  display: block;
+  float: left;
+  color: #0c145e;
+  width: 50px;
+  cursor: pointer;
+  padding-top: 2px;
+  padding-left: 10px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] **[OPTIONAL]** I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex or any other property:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓